loklak.php

```diff
@@ -230,4 +230,45 @@ public function aggregations($query="", $since=null, $until=null, $fields=null,
 			return json_encode($request, true);
 		}
 	}
+
+	public function account($name, $action=null, $data=null) {
+		$this->requestURL = 'http://localhost:9000/api/account.json';
+		$this->name = $name;
+		$this->action = $action;
+		$this->data = $data;
+		$headers = array();
+		$headers['User-Agent'] = "Mozilla/5.0 (Android 4.4; Mobile; rv:41.0) Gecko/41.0 Firefox/41.0";
+		$headers['From'] = "info@loklak.org";
+		if ($name) {
+			$params = array('screen_name'=>$this->name);
+			$request = Requests::request($this->requestURL, array('Accept' => 'application.json'), $headers, $params);
```

Comment:
Please see that the format of request is acceptable with http://requests.ryanmccue.info/api/source-class-Requests.html#246-322